Error when starting docker-make : declaration of "err" shadows declaration at line 98

**Mattermost Version**
/

**Plugin Version**
Lattest (from Git : 7c04448d7e9dd4bc972bc2364e4b816b579b6c8f)

**Describe the issue**
After cloning the project, nothing has been changed on the files, starting ./docker-maker : it stops with this error : 

> Running govet
> env GO111MODULE=off /usr/local/go/bin/go get golang.org/x/tools/go/analysis/passes/shadow/cmd/shadow
> /usr/local/go/bin/go vet ./server/...
> /usr/local/go/bin/go vet -vettool=/go/bin/shadow ./server/...
> # github.com/mattermost/mattermost-plugin-demo/server
> server/plugin.go:104:8: declaration of "err" shadows declaration at line 98
> server/plugin.go:170:8: declaration of "err" shadows declaration at line 98
> make: *** [Makefile:60: govet] Error 2


**Additional context**
I runned the same code a few days ago and everything worked fine. Has "#github.com/mattermost/mattermost-plugin-demo/server" repository changed in between ?

--- a/server/plugin.go
+++ b/server/plugin.go
@@ -101,7 +101,7 @@ func (p *Plugin) handleStartMeeting(w http.ResponseWriter, r *http.Request) {
 		http.Error(w, err.Error(), err.StatusCode)
 	}
 
-	if _, err := p.API.GetChannelMember(req.ChannelId, user.Id); err != nil {
+	if _, err = p.API.GetChannelMember(req.ChannelId, user.Id); err != nil {
 		http.Error(w, "Forbidden", http.StatusForbidden)
 		return
 	}
@@ -167,7 +167,7 @@ func (p *Plugin) handleStartMeeting(w http.ResponseWriter, r *http.Request) {
 		},
 	}
 
-	if _, err := p.API.CreatePost(post); err != nil {
+	if _, err = p.API.CreatePost(post); err != nil {
 		http.Error(w, err.Error(), err.StatusCode)
 		return
 	}

